runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -10806,6 +10806,172 @@ inlineMathSQRT(
 
    }
 
+static bool inlineMathFMAFloat(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   // fma(a, b, c) => (a*b+c)
+   TR::Node *firstChild = node->getChild(0);
+   TR::Node *secondChild = node->getChild(1);
+   TR::Node *thirdChild = node->getChild(2);
+   
+   TR::Register *targetRegister = NULL;
+
+   if(firstChild->getOpCode().isLoadConst() && 
```

Comment:
This case should not occur - the simplifier in the compiler should have folded the result prior to this point.

Reply:
Shifted constant folding to J9Simplifier.cpp